behat.yml.dist

```diff
@@ -59,6 +60,18 @@ default:
                 firefox:
                     selenium2:
                         browser: firefox
+                panther:
+                    panther:
+                        manager_options:
+                            connection_timeout_in_ms: 5000
+                            request_timeout_in_ms: 120000
+                            chromedriver_arguments:
+                                - --log-path=etc/build/chromedriver.log
+                                - --verbose
+                            capabilities:
+                                acceptSslCerts: true
+                                acceptInsecureCerts: true
+                                unexpectedAlertBehaviour: accept
```

Comment:
Eratum: I previously comented this part and deleted this comment, I copied an old version of my panther config, I'm using this exact same config so no change requested here.